Fotogjengen/hilfling-photo-backend
# Feature/taheera/soft deletes

Fixed soft deletes for tables that extends the base repository. 

How to test: 
1. Run the script `./dropreacreatedb.sh` and seed the db
2. Send a GET-request to find an ID like `localhost:8080/albums`
3. Send a DELETE-request to `localhost:8080/albums/{ID}`
4. Send a new GET-request to `localhost:8080/albums`
5. The entry you deleted in step 3 should be gone now B)
